src/pages/Send/components/RecipientPicker/WalletList/WalletList.js

```diff
@@ -37,7 +39,7 @@ function getLineCount(wallets: RecipientPickerWallet[]) {
     // eslint-disable-next-line fp/no-mutation
     num += 1
 
-    if (wallet.addresses && wallet.addresses.length > 1) {
+    if (wallet.addresses && wallet.type === 'mnemonic') {
```

Comment:
Better to use `utils/wallets/checkMultiAddressType`
Please, take it to account in future